zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java

```diff
@@ -904,9 +904,11 @@ public String getStatus(String sessionId, String jobId)
       }
 
       for (Interpreter intp : interpreters) {
-        Job job = intp.getScheduler().getJob(jobId);
-        if (job != null) {
-          return job.getStatus().name();
+        if (((LazyOpenInterpreter) intp).isOpen()) {
```

Comment:
At the moment I can't view the whole source code. A `instanceof` should avoid a `ClassCastException`.

Reply:
@Reamer I use another approach which don't need class cast